ambry-api/src/main/java/com.github.ambry/config/FrontendConfig.java

```diff
@@ -50,6 +50,13 @@
   @Default("")
   public final List<String> frontendPathPrefixesToRemove;
 
+  /**
+   * Specifies the blob size in bytes beyond which an HTTP Chunked response is used to send response getBlob response.
+   */
+  @Config("chunkedGetResponseThresholdInBytes")
```

Comment:
also prefix with frontend. please be consistent with format already in the file.